diff --git a/node_modules/expo/build/deprecatedGlobal.js b/node_modules/expo/build/deprecatedGlobal.js
index 83c9a47..50d8a55 100644
--- a/node_modules/expo/build/deprecatedGlobal.js
+++ b/node_modules/expo/build/deprecatedGlobal.js
@@ -15,7 +15,7 @@ function logWarning() {
     instructions += packages.join(', ');
     instructions += `.\n\n`;
     instructions += `The global "__expo" and "Expo" objects will be removed in SDK 41. Learn more about how to fix this warning: https://expo.fyi/deprecated-globals\n`;
-    console.warn(`Your project is accessing the following APIs from a deprecated global rather than a module import: ${instructions}`);
+    //console.warn(`Your project is accessing the following APIs from a deprecated global rather than a module import: ${instructions}`);
     packages = [];
 }
 //# sourceMappingURL=deprecatedGlobal.js.map
\ No newline at end of file
