diff --git a/node_modules/react-native-elements/src/list/ListItem.js b/node_modules/react-native-elements/src/list/ListItem.js
index ce94447..a794353 100644
--- a/node_modules/react-native-elements/src/list/ListItem.js
+++ b/node_modules/react-native-elements/src/list/ListItem.js
@@ -38,7 +38,7 @@ const renderAvatar = (content) =>
 
 const renderIcon = (content, theme) =>
   renderNode(Icon, content, {
-    color: Platform.select(theme.colors.platform).grey,
+    //color: Platform.select(theme.colors.platform).grey,
     size: 24,
   });
 
@@ -89,126 +89,126 @@ const ListItem = (props) => {
     ...attributes
   } = props;
 
-  if (title) {
-    console.warn(
-      "'ListItem.title' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (titleStyle) {
-    console.warn(
-      "'ListItem.titleStyle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (titleProps) {
-    console.warn(
-      "'ListItem.titleProps' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (subtitle) {
-    console.warn(
-      "'ListItem.subtitle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (subtitleStyle) {
-    console.warn(
-      "'ListItem.subtitleStyle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (subtitleProps) {
-    console.warn(
-      "'ListItem.subtitleProps' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (leftIcon) {
-    console.warn(
-      "'ListItem.leftIcon' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (leftAvatar) {
-    console.warn(
-      "'ListItem.leftAvatar' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (leftElement) {
-    console.warn(
-      "'ListItem.leftElement' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (rightTitle) {
-    console.warn(
-      "'ListItem.rightTitle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (rightTitleStyle) {
-    console.warn(
-      "'ListItem.rightTitleStyle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (rightTitleProps) {
-    console.warn(
-      "'ListItem.rightTitleProps' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (rightSubtitle) {
-    console.warn(
-      "'ListItem.rightSubtitle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (rightSubtitleStyle) {
-    console.warn(
-      "'ListItem.rightSubtitleStyle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (rightSubtitleProps) {
-    console.warn(
-      "'ListItem.rightSubtitleProps' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (input) {
-    console.warn(
-      "'ListItem.input' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (buttonGroup) {
-    console.warn(
-      "'ListItem.buttonGroup' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (switchProps) {
-    console.warn(
-      "'ListItem.switch' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (checkBox) {
-    console.warn(
-      "'ListItem.checkBox' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (badge) {
-    console.warn(
-      "'ListItem.badge' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (chevron) {
-    console.warn(
-      "'ListItem.chevron' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (contentContainerStyle) {
-    console.warn(
-      "'ListItem.contentContainerStyle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (rightContentContainerStyle) {
-    console.warn(
-      "'ListItem.rightContentContainerStyle' prop has been deprecated and will be removed in the next version."
-    );
-  }
-  if (checkmark) {
-    console.warn(
-      "'ListItem.checkmark' prop has been deprecated and will be removed in the next version."
-    );
-  }
+  // if (title) {
+  //   console.warn(
+  //     "'ListItem.title' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (titleStyle) {
+  //   console.warn(
+  //     "'ListItem.titleStyle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (titleProps) {
+  //   console.warn(
+  //     "'ListItem.titleProps' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (subtitle) {
+  //   console.warn(
+  //     "'ListItem.subtitle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (subtitleStyle) {
+  //   console.warn(
+  //     "'ListItem.subtitleStyle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (subtitleProps) {
+  //   console.warn(
+  //     "'ListItem.subtitleProps' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (leftIcon) {
+  //   console.warn(
+  //     "'ListItem.leftIcon' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (leftAvatar) {
+  //   console.warn(
+  //     "'ListItem.leftAvatar' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (leftElement) {
+  //   console.warn(
+  //     "'ListItem.leftElement' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (rightTitle) {
+  //   console.warn(
+  //     "'ListItem.rightTitle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (rightTitleStyle) {
+  //   console.warn(
+  //     "'ListItem.rightTitleStyle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (rightTitleProps) {
+  //   console.warn(
+  //     "'ListItem.rightTitleProps' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (rightSubtitle) {
+  //   console.warn(
+  //     "'ListItem.rightSubtitle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (rightSubtitleStyle) {
+  //   console.warn(
+  //     "'ListItem.rightSubtitleStyle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (rightSubtitleProps) {
+  //   console.warn(
+  //     "'ListItem.rightSubtitleProps' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (input) {
+  //   console.warn(
+  //     "'ListItem.input' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (buttonGroup) {
+  //   console.warn(
+  //     "'ListItem.buttonGroup' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (switchProps) {
+  //   console.warn(
+  //     "'ListItem.switch' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (checkBox) {
+  //   console.warn(
+  //     "'ListItem.checkBox' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (badge) {
+  //   console.warn(
+  //     "'ListItem.badge' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (chevron) {
+  //   console.warn(
+  //     "'ListItem.chevron' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (contentContainerStyle) {
+  //   console.warn(
+  //     "'ListItem.contentContainerStyle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (rightContentContainerStyle) {
+  //   console.warn(
+  //     "'ListItem.rightContentContainerStyle' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
+  // if (checkmark) {
+  //   console.warn(
+  //     "'ListItem.checkmark' prop has been deprecated and will be removed in the next version."
+  //   );
+  // }
 
   return (
     <Component
diff --git a/node_modules/react-native-elements/src/list/ListItemSubtitle.js b/node_modules/react-native-elements/src/list/ListItemSubtitle.js
index 482a41e..8fd463f 100644
--- a/node_modules/react-native-elements/src/list/ListItemSubtitle.js
+++ b/node_modules/react-native-elements/src/list/ListItemSubtitle.js
@@ -30,14 +30,14 @@ const styles = {
         fontSize: 15,
       },
       default: {
-        color: ANDROID_SECONDARY,
+        //color: ANDROID_SECONDARY,
         fontSize: 14,
       },
     }),
   },
 
   rightSubtitle: {
-    color: ANDROID_SECONDARY,
+    //color: ANDROID_SECONDARY,
   },
 };
 
